.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: /
+  schedule:
+    interval: daily
```

Comment:
This will be noop :-)
because we do not have pinned versions in requirments*.txt
* https://github.com/release-engineering/pubtools-marketplacesvm/blob/2592e328506bd4d3f1f0591b9ef1df7b2e23cb4c/requirements-test.txt
* https://github.com/release-engineering/pubtools-marketplacesvm/blob/2592e328506bd4d3f1f0591b9ef1df7b2e23cb4c/requirements.txt